pkg/bkctl/ledger/delete_test.go

```diff
@@ -23,6 +23,15 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestDeleteCmd(t *testing.T) {
+	args := []string{"delete", "1"}
```

Comment:
why deleting the ledger id "1" should always succeed here?

Reply:
Delete a non-existent ledger will return success and print a warning log.